app/src/main/java/io/intrepid/contest/screens/splash/SplashActivity.java

```diff
@@ -25,6 +35,18 @@ protected int getLayoutResourceId() {
         return R.layout.activity_splash;
     }
 
+    @Override
+    public void showCreateContestScreen() {
+        UUID userId = TestConfig.userid;
+        Intent intent = NewContestActivity.createIntent(this, userId);
+        startActivity(intent);
+    }
+
+    @Override
+    public void showJoinContestScreen() {
+        Timber.d("Create contest clicked");
```

Comment:
If you want you can return a toast here too to make it easier to test without a computer connect. Not a requirement though.